hathor/consensus/block_consensus.py

```diff
@@ -200,7 +201,7 @@ def update_voided_info(self, block: Block) -> None:
                 common_block = self._find_first_parent_in_best_chain(block)
                 self.add_voided_by_to_multiple_chains(block, heads, common_block)
 
-                if score >= best_score + self._settings.WEIGHT_TOL:
+                if score > best_score:
```

Comment:
Why was the `=` removed?

Reply:
Here we are checking if the block has the highest score. So we must check if `score > best_score`.

@glevco and @jansegre , confirm that you agree that I did the right thing. I'll wait for your confirmation before resolving this conversation.

Reply:
I agree.

Reply:
It seems correct to me so I've kept it.